modules/drivers/sparksql/src/metabase/driver/sparksql.clj

```diff
@@ -105,7 +105,7 @@
                                               (format "describe `%s`.`%s`"
                                                       (dash-to-underscore (:schema table))
                                                       (dash-to-underscore (:name table)))
-                                              (str "describe " (dash-to-underscore (:name table))))])]
+                                              (str "describe " (dash-to-underscore (:name table))))]) :while (and (not(clojure.string/blank? (:col_name result))) (not(clojure.string/blank? (:data_type result))))]
```

Comment:
Hi, 
mind formatting this code according to the [Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide)?